power/Power.cc

```diff
@@ -1204,7 +1204,14 @@ Power::findActivity(const Pin *pin)
     PwrActivity &activity = activity_map_[pin];
     if (activity.origin() != PwrActivityOrigin::unknown)
       return activity;
+  } else if (user_activity_map_.hasKey(pin)) {
```

Comment:
This annotates the net activity on to the pin, right? Just for my understanding.

Reply:
The activity_map_ used originally does not contain all the pins, on the other hand, the user_activity_map_ does contain all the pins and is needed to be able to calculate the Power on all the cells. I found that through gdb debugging. Maybe the activity_map_ was meant to be  user_activity_map_, something to check with Cherry.